tests/SprykerEcoTest/Zed/UnzerApi/Business/UnzerApiFacade/PerformUpdateCustomerApiCallTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\UnzerApi\Business\UnzerApiFacade;
+
+use Generated\Shared\Transfer\UnzerApiErrorResponseTransfer;
+use SprykerEcoTest\Zed\UnzerApi\Business\UnzerApiFacadeBaseTest;
+
+/**
+ * @group Functional
```

Comment:
annotations for tests should be regenerated
I believe we should have `UnzerApiFacadeBaseTest` group here